async-request-reply/README.md

```diff
@@ -84,21 +111,58 @@ For more information about this pattern, see [Asynchronous Request-Reply pattern
 1. Delete all the Azure resources for this Cloud Async Pattern
 
    ```bash
-   az group delete -n rg-asyncrequestreply -y
+   az group delete -n ${RESOURCEGROUP} -y
    ```
```

Comment:
~~~suggestion
Most of the Azure resources deployed in the prior steps will incur ongoing charges unless removed.

```bash
az group delete -n ${RESOURCEGROUP} -y
```
~~~